crates/breez-sdk/core/src/nostr.rs

```diff
@@ -59,11 +39,14 @@ impl NostrClient {
             ));
         };
 
-        // Convert bitcoin SecretKey to nostr SecretKey
-        let keys = Keys::new(self.nostr_key.clone());
+        let builder =
+            lnurl_models::nostr::create_zap_receipt(zap_request, invoice, preimage.clone())
+                .map_err(NostrError::ZapReceiptCreationError)?;
```

Comment:
Nit, but IMO helps with readability
```suggestion
        let builder =
            lnurl_models::nostr::create_zap_receipt_event_builder(zap_request, invoice, preimage.clone())
                .map_err(NostrError::ZapReceiptCreationError)?;
```

Reply:
Yes. But just nitpicking. Also fine to keep as is